lib/stones-spec/language/gobstones.rb

```diff
@@ -24,10 +24,14 @@ def self.parse_error_message(result)
       "<pre>#{ErrorMessageParser.new.parse(result)}</pre>"
     end
 
-    def self.is_runtime_error?(result)
+    def self.runtime_error?(result)
       result.include? 'Error en tiempo de ejecución'
     end
 
+    def self.syntax_error?(result)
+      result.include_any? ['Error de sintaxis', 'Error en el programa']
```

Comment:
It was already defined at the top of gobstones.rb :P
El 21/9/2015 18:07, "Federico Aloi" <notifications@github.com> escribió:

> In lib/stones-spec/language/gobstones.rb
> <https://github.com/uqbar-project/stones-spec/pull/29#discussion_r40026189>
> :
>
> >        result.include? 'Error en tiempo de ejecución'
> >      end
> >
> > +    def self.syntax_error?(result)
> > +      result.include_any? ['Error de sintaxis', 'Error en el programa']
>
> Where is include_any? defined?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uqbar-project/stones-spec/pull/29/files#r40026189>.
>